diff -Naur cmake-3.16.5/Modules/FindRuby.cmake cmake-3.16.5_new/Modules/FindRuby.cmake
--- cmake-3.16.5/Modules/FindRuby.cmake	2020-03-04 21:50:44.000000000 +0800
+++ cmake-3.16.5_new/Modules/FindRuby.cmake	2020-07-28 21:08:42.656683293 +0800
@@ -109,15 +109,8 @@
   # site_ruby
   _RUBY_CONFIG_VAR("sitearchdir" RUBY_SITEARCH_DIR)
   _RUBY_CONFIG_VAR("sitelibdir" RUBY_SITELIB_DIR)
-
-  # vendor_ruby available ?
-  execute_process(COMMAND ${RUBY_EXECUTABLE} -r vendor-specific -e "print 'true'"
-    OUTPUT_VARIABLE RUBY_HAS_VENDOR_RUBY  ERROR_QUIET)
-
-  if(RUBY_HAS_VENDOR_RUBY)
-    _RUBY_CONFIG_VAR("vendorlibdir" RUBY_VENDORLIB_DIR)
-    _RUBY_CONFIG_VAR("vendorarchdir" RUBY_VENDORARCH_DIR)
-  endif()
+  _RUBY_CONFIG_VAR("vendorlibdir" RUBY_VENDORLIB_DIR)
+  _RUBY_CONFIG_VAR("vendorarchdir" RUBY_VENDORARCH_DIR)
 
   # save the results in the cache so we don't have to run ruby the next time again
   set(RUBY_VERSION_MAJOR    ${RUBY_VERSION_MAJOR}    CACHE PATH "The Ruby major version" FORCE)
